app/views/liquids/_twitch.html.erb

```diff
@@ -1,6 +1,8 @@
 <iframe
-    src="https://clips.twitch.tv/embed?autoplay=false&clip=<%= slug %>&parent=<%= url %>"
+    src="<%= url %>"
```

Comment:
I decided to pass the entire URL in as a local because the URL is different for Twitch videos and clips, and I did not want to do too much logic/manipulation within the view.